.gitlab-ci.yml

```diff
@@ -0,0 +1,16 @@
+# See https://docs.gitlab.com/ce/ci/yaml/README.html for the documentation
+
+package:
+  image: debian:10
+  stage: build
+  before_script:
+    - apt update
+    - apt install --yes python3-pip
+    - pip3 install -r requirements.txt
+  script:
+    - ./build-documentation.sh
+  artifacts:
+    when: always
+    paths:
+      - sphinx-output.log
+      - build
```

Comment:
Can you check how big is the build directory? I'm assuming it's tens or hundreds of megabytes. If yes, consider gzipping that to save some disk space and bandwidth on upload.

Reply:
But if we zip it we loose the funtionality to browse the files directly. Selecting browse from https://gitlab.com/t-b/tango-doc/-/jobs/965595694 leads to https://t-b.gitlab.io/-/tango-doc/-/jobs/965595694/artifacts/build/overview/index.html which allows to quickly look at the result.

The size differences are:
$ du -sh build build.gz
140M    build
102M    build.gz

Reply:
Got it. Thanks for checking! Probably pictures contribute a lot to the output size, thus there is no big difference in compressed and uncompressed sizes. What you did is indeed better.